src/TestFramework/PhpUnit/Config/Builder/MutationConfigBuilder.php

```diff
@@ -88,9 +91,7 @@ public function build(
         string $mutationHash,
         string $mutationOriginalFilePath
     ): string {
-        // clone the dom document because it's mutated later
-        $dom = clone $this->dom;
```

Comment:
Could not find evidence that the `clone` was necessary here. Would be happy to add it back if we can find a test capturing this scenario

Reply:
So it is important, at least that's why the build is failing. I'm however unable to find a way to capture this issue, does someone has an idea?

Reply:
Found it